cosmic_ray/cli.py

```diff
@@ -56,7 +56,7 @@ def handle_help(config):
     if not command:
         options = OPTIONS
     elif command not in COMMAND_HANDLER_MAP:
-        LOG.error('"{}" is not a valid cosmic-ray command'.format(command))
+        LOG.error('"%s" is not a valid cosmic-ray command' % command)
```

Comment:
Is this the warning you're seeing?

> `W:102,10: Use % formatting in logging functions and pass the % parameters as arguments (logging-format-interpolation)`

If so, I'm certain that they're asking us to pass the interpolation arguments as arguments to the logging function, and not to do string interpolation with `% or `format()`. Passing them as arguments is certainly a best practice, and it looks like the [pylint pull request for this warning](https://bitbucket.org/logilab/pylint/pull-requests/169/add-new-warning-logging-format/diff) was intended to get people to use the argument-passing form. 

I agree, though, that the language of the warning is a bit confusing. It really does sound like they want you to pass the arguments using `%`.